packages/contracts/script/DeploymentConfig.s.sol

```diff
@@ -112,6 +118,24 @@ contract DeploymentConfig is Script {
         });
     }
 
+    function getOptimismGoerliConfig() public returns (NetworkConfig memory) {
```

Comment:
Non-blocking: would remove `getMainnetConfig` and `getGoerliEthConfig` since not used.

Reply:
Fair enough. Will do in a follow-up PR.

Reply:
https://github.com/status-im/community-dapp/pull/99